File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        int a = 0;
        int b = 0;
        
        if(arr1.length > arr2.length){
            answer = 1;
        }else if(arr1.length < arr2.length){
            answer = -1;
        }else{
            for(int i=0;i<arr1.length;i++){
                a += arr1[i];
                b += arr2[i];
            }
            answer = a == b ? 0 : a > b ? 1 : -1;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    a = 0
    b = 0
    
    if len(arr1) > len(arr2):
        answer = 1;
    elif len(arr1) < len(arr2):
        answer = -1;
    else:
        for i in range(len(arr1)):
            a += arr1[i];
            b += arr2[i];
        if a > b:
            answer = 1;
        elif a < b:
            answer = -1;
        else:
            answer = 0;
    
    return answer